apps/bttester/src/btp_bap.c

```diff
@@ -319,6 +329,74 @@ broadcast_source_start(const void *cmd, uint16_t cmd_len, void *rsp,
     return BTP_STATUS_SUCCESS;
 }
 
+static uint8_t
+broadcast_code_set(const void *cmd, uint16_t cmd_len, void *rsp,
+                   uint16_t *rsp_len)
+{
+    int i;
+    const struct btp_bap_set_broadcast_code_cmd *cp = cmd;
+
+    sinks[sink_num].addr = cp->addr;
```

Comment:
we should reject if sink_num is too big,  now it will write pass sinks[]

Reply:
Done